.github/workflows/explorer-dev.yaml

```diff
@@ -0,0 +1,21 @@
+name: Enable Explorer maintenance on DEV
+on:
+  workflow_dispatch:
+
+jobs:
+  build_and_deploy_explorer:
+    uses: Cerebellum-Network/reusable-workflows/.github/workflows/build-and-upload-static.yaml@1.0.0
+    with:
+      runs-on: '["self-hosted", "cere-network-large-workers"]'
+      build_container: 'node:14-buster-slim'
+      deploy_container: 'ubuntu:20.04'
+      install_packages_command: 'echo "Installing is not required"'
+      build_command: 'printf "Cere team\n team@cere.network\n" | ./install.sh'
+      path_to_static_files_to_upload: './dist'
+    secrets:
+      NETWORK_AWS_ACCESS_KEY_ID: ${{ secrets.DEV_NETWORK_AWS_ACCESS_KEY_ID }}
+      NETWORK_AWS_SECRET_ACCESS_KEY: ${{ secrets.DEV_NETWORK_AWS_SECRET_ACCESS_KEY }}
+      AWS_REGION: ${{ secrets.AWS_REGION }}
+      S3_BUCKET_NAME: ${{ secrets.EXPLORER_S3_BUCKET_NAME_DEV }}
```

Comment:
Could we use `DEV` always at the beginning or at the end?

Reply:
At the end seems  consistent with the previous naming convention.